README.md

```diff
@@ -83,10 +83,30 @@ variables:
       (setf (alist-get 'black apheleia-formatters)
             '("black" "--option" "..." "-"))
       ```
-
 * `apheleia-mode-alist`: Alist mapping major modes and filename
   regexps to names of formatters to use in those modes and files. See
   the docstring for more information.
+    * You can use this variable to configure multiple formatters for
+      the same buffer by setting the `cdr` of an entry to a list of
+      formatters to run instead of a single formatter. For example you
+      may want to run `isort` and `black` one after the other.
+
+      ```elisp
+      (push '(isort . ("isort" "--stdout" "-"))
+            apheleia-formatters)
```

Comment:
Would it make sense to use `(setf (alist-get 'isort apheleia-mode-alist) ...)` here as well?

Reply:
I suppose so, I wasn't aware setf actually pushed new entries onto the list as well. I thought if alist-get returns nil it would just fail but I just tested it out and it works like it should. In the meantime what're your thoughts on adding isort to `apheleia-formatters`?